api/oauth/oauth.go

```diff
@@ -22,17 +21,36 @@ func NewService() *Service {
 	return &Service{}
 }
 
+type cesUserData struct {
```

Comment:
We should refrain from using CES directly in this code when these are just regular LDAP attributes.

Reply:
AuthenticationData / AuthenticationUserData